pkg/core/interop/contract/call.go

```diff
@@ -32,7 +32,7 @@ func LoadToken(ic *interop.Context) func(id int32) error {
 		}
 		cs, err := ic.GetContract(tok.Hash)
 		if err != nil {
-			return fmt.Errorf("contract not found: %w", err)
+			return fmt.Errorf("token contract %s not found: %w", tok.Hash.StringLE(), err)
```

Comment:
It's not about ` rpc: use non-native NS instead of NNS in testchain `.

Reply:
Moved to a separate commit.